javascript/javascript3/week2/currency-calculator/app.js

```diff
@@ -0,0 +1,103 @@
+// Currency converter
+
+// Variables
+const url1 = "https://v6.exchangerate-api.com/v6/ece1faae718b5e53f9a037c5/latest/EUR/";
+const url2 = "https://v6.exchangerate-api.com/v6/ece1faae718b5e53f9a037c5/latest/DKK/";
+const selectCurrencyFrom = document.getElementById("currency-from-selector");
+const selectCurrencyTo = document.getElementById("currency-to-selector");
+const currencyFrom = document.getElementById("from");
+const currencyTo = document.getElementById("to");
+const lastUpdateDate = document.getElementById("exchange-date");
+
+// Fetching the data
+async function fetchCurrency() {
+  try {
+    // Fetch both URLs
+    const [data1, data2] = await Promise.all([fetchData(url1), fetchData(url2)]);
+
+    // Store conversion rates
+    conversionRates = data1.conversion_rates;
+
+    getCurrencies(data1.conversion_rates, selectCurrencyFrom);
+    getCurrencies(data2.conversion_rates, selectCurrencyTo);
+
+    // Get the last updated data
+    let formattedTimeUtc = data1.time_last_update_utc.replace(/\+0000$/, "");
+    lastUpdateDate.innerHTML = formattedTimeUtc;
+
+    console.log(data1, data2);
+  } catch (error) {
+    console.log(`Error: ${error.message}`);
+  }
+}
+
+fetchCurrency();
```

Comment:
`fetchCurrency` is an `async` function so you need to await it.

```suggestion
await fetchCurrency();
```